### PR TITLE
Add frame-resize option to ssd demo to scale camera input

### PR DESCRIPTION
**Pull request by larroy**
## Description ##
Minor pr. added rescaling of webcam input

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
 
@zhreshold

### CODE REVIEW

**Comment by zhreshold** [comment]
CI is down.

File: example/ssd/dataset/cv2Iterator.py [hunk 1]
```diff
@@ -26,10 +26,18 @@ class CameraIterator():
...
+                self._frame_resize = tuple(map(int, frame_resize))
+                self._frame_shape = (1, 3, self._frame_resize[0], self._frame_resize[1])
+            elif isinstance(frame_resize, float):
+                width = int(self._capture.get(cv2.CAP_PROP_FRAME_WIDTH)*frame_resize)
+                height = int(self._capture.get(cv2.CAP_PROP_FRAME_HEIGHT)*frame_resize)
+                self._frame_shape = (1, 3, width, height)
+                self._frame_resize = (width, height)
+            else:
+                assert False, "frame_resize should be a tuple of (x,y) pixels "
+                "or an float setting the scaling factor"
```
Comment (lebeg) [inline example/ssd/dataset/cv2Iterator.py]: a float, not an float

File: example/ssd/demo.py [hunk 2]
```diff
@@ -109,6 +109,8 @@ def parse_args():
                         help='string of comma separated names, or text filename')
     parser.add_argument('--camera', action='store_true',
                         help="use camera for image capturing")
+    parser.add_argument('--frame-resize', type=str, default=None
```
Comment (lebeg) [inline example/ssd/demo.py]: should this default to 1.0?

Comment (zhreshold) [reply example/ssd/demo.py]: None is fine

**Comment by zhreshold** [comment]
@larroy Rebase to fix CI?

**Comment by larroy** [comment]
@zhreshold  done

File: example/ssd/demo.py (hunk 2 above)
Comment (larroy) [reply example/ssd/demo.py]: We avoid a call to resizing if it's none

**Pull request merged**